src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/orgRules.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {css, ClassNames} from '@emotion/core';
+
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import {IconChevron} from 'app/icons';
+
+import RulesList from './rulesList';
+import {Rule} from './types';
+
+type Props = {
+  rules: Array<Rule>;
+};
+
+type State = {
+  isCollapsed: boolean;
+  contentHeight?: string;
+};
+
+class OrgRules extends React.Component<Props, State> {
+  state: State = {
+    isCollapsed: true,
+  };
+
+  componentDidUpdate() {
+    this.loadContentHeight();
+  }
+
+  rulesListRef = React.createRef<HTMLUListElement>();
+
+  loadContentHeight = () => {
+    if (!this.state.contentHeight) {
+      const contentHeight = this.rulesListRef.current?.offsetHeight;
+      if (contentHeight) {
+        this.setState({
+          contentHeight: `${contentHeight}px`,
+        });
+      }
+    }
+  };
+
+  handleToggleCollapsed = () => {
+    this.setState(prevState => ({
+      isCollapsed: !prevState.isCollapsed,
+    }));
+  };
+
+  render() {
+    const {rules} = this.props;
+    const {isCollapsed, contentHeight} = this.state;
+
+    if (rules.length === 0) {
+      return (
+        <Wrapper>{t('There is no data privacy rules in the organization level')}</Wrapper>
+      );
+    }
+    return (
+      <ClassNames>
+        {({css: classNamesCss}) => (
+          <Wrapper isCollapsed={isCollapsed} contentHeight={contentHeight}>
+            <Header onClick={this.handleToggleCollapsed}>
+              <div>{t('Organization Rules')}</div>
+              <Button
+                icon={<IconChevron size="xs" direction={isCollapsed ? 'down' : 'up'} />}
+                size="xsmall"
+                labelClassName={classNamesCss`
+                  padding: ${space(0.25)} ${space(0.5)};
+                `}
+              />
```

Comment:
Let's add a `label` here since it's only an icon.

Reply:
please do you mean title?